### PR TITLE
Create new branch

### PR DESCRIPTION
**Pull request by letcheep**
creat new branch

### CODE REVIEW

**Comment by l2el3er** [comment]
Test comment

File: how-to-user-git.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+How to use Git
+1.Clone git repro to your machine
```
Comment (l2el3er) [inline how-to-user-git.txt:2]: Repo

Comment (letcheep) [reply how-to-user-git.txt:2]: ok

Comment (letcheep) [reply how-to-user-git.txt]: updated

File: how-to-user-git.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+How to use Git
+1.Clone git repo to your machine
+2.Create new branch by " Create brance from... "
```
Comment (l2el3er) [inline how-to-user-git.txt:3]:
```suggestion
2.Create new branch by " Create branch from... " 
```
fix misspelled

**Review by l2el3er — APPROVED** [review]

**Pull request merged**